TLM/TLM/Util/Extensions/VehicleExtensions.cs

```diff
@@ -36,20 +36,23 @@ public static class VehicleExtensions {
             vehicle.m_flags.IsFlagSet(Vehicle.Flags.WaitingPath);
 
         /// <summary>Determines the <see cref="ExtVehicleType"/> for a vehicle.</summary>
```

Comment:
> Determines the `ExtVehicleType` for a vehicle.

No longer true since it is no longer using `DetermineVehicleTypeFromAIType()`

Reply:
code reverted to orignal approach that identifies via vehicle ai